R/MoBiProject.R

```diff
@@ -0,0 +1,163 @@
+#' @title MoBi Project
+#' @docType class
+#' @description  A MoBi project, containing modules, building blocks, simulations, etc.
+#' @format NULL
+MoBiProject <- R6::R6Class(
+  "MoBiProject",
+  cloneable = FALSE,
+  inherit = ObjectBase,
+  active = list(
+    #' @field sourceFile Path to the file the project was loaded from (read-only)
+    sourceFile = function(value) {
+      private$.readOnlyProperty("sourceFile", value, private$.sourceFile)
+    },
+    #' @field simulationNames Names of the simulations that are present in the project
+    simulationNames = function(value) {
+      if (missing(value)) {
+        model <- self$get("SimulationNames")
+      } else {
+        private$.throwPropertyIsReadonly("simulationNames")
+      }
+    },
+    #' @field parameterIdentificationNames Names of the parameter identifications that are present in the project
+    #'
+    #' PRIO 2
+    parameterIdentificationNames = function(value) {
+      if (missing(value)) {
+        model <- self$get("ParameterIdentificationNames")
+      } else {
+        private$.throwPropertyIsReadonly("parameterIdentificationNames")
+      }
+    },
+    #' @field Individuals Names of the individuals that are present in the project
+    Individuals = function(value) {
```

Comment:
Consistency: `Individuals `and `ExpressionProfiles `start with the upper case letter, while other fields start with the lower case